src/components/button/button.scss

```diff
@@ -229,15 +229,16 @@
 }
 
 // button styles
-// solid
-:host([appearance="solid"]) {
+:host([appearance="solid"]),
+:host([appearance]) {
```

Comment:
I'm not sure why we need the `:host([appearance="solid"]),` if `:host([appearance]) {` covers all cases right?